### DISCUSSION TITLE
Set random_state value for RandomForestClassifier ?

### DISCUSSION THREAD

**Discussion by StefanTroost** [asreview/asreview#722] [Ideas]
When looking at the API for Random Forest classifier, I noticed that RF doesn't have a random state by default so you would have to set it and report it? For reproducibility purposes I mean ?

**Comment by Rensvandeschoot** [comment]
Dear @StefanTroost 

Thank you for your question! 

You are referring to the default settings of the Random Forest Classifier as explained in [the documentation](https://asreview.readthedocs.io/en/latest/API/reference.html?highlight=random%20forest#asreview.models.classifiers.RandomForestClassifier) of which the relevant part is:


```
•	random_state (int or RandomState, default=None) – Controls both the randomness of the bootstrapping of the samples used when building trees and the sampling of the features to consider when looking for the best split at each node.

```

The `default=None` refers to random seed values which are used. 

If you want to set a specific seed value you can use `--seed ` via the CLI. 

Hope this answered your question!
